jenkins_jobs/build_host_os/script.sh

```diff
@@ -1,5 +1,5 @@
 # ISO 8601 date with nanoseconds precision
-TIMESTAMP=$(date --utc -Ins)
+TIMESTAMP=$(date --utc +'%Y-%m-%dT%H:%M:%S:%N+%Z')
```

Comment:
I tested with `2017-02-02T11:43:17.134075170+0000` and it worked just fine from yum perspective.